vmm/src/lib.rs

```diff
@@ -998,7 +1017,37 @@ impl Vmm {
             }
 
             // Send config
-            let config_data = serde_json::to_vec(&vm.get_config()).unwrap();
+            let vm_config = vm.get_config();
+            #[cfg(all(feature = "kvm", target_arch = "x86_64"))]
+            let common_cpuid = {
+                #[cfg(feature = "tdx")]
+                let tdx_enabled = vm_config.lock().unwrap().tdx.is_some();
+                let phys_bits = vm::physical_bits(
+                    vm_config.lock().unwrap().cpus.max_phys_bits,
+                    #[cfg(feature = "tdx")]
+                    tdx_enabled,
+                );
+                arch::generate_common_cpuid(
+                    self.hypervisor.clone(),
+                    None,
+                    None,
+                    phys_bits,
+                    vm_config.lock().unwrap().cpus.kvm_hyperv,
+                    #[cfg(feature = "tdx")]
+                    tdx_enabled,
+                )
+                .map_err(|_| {
+                    MigratableError::MigrateReceive(anyhow!("Error 'generate_common_cpuid'"))
```

Comment:
This is a strange error message.

Reply:
Good catch. Forgot to fix the error handling. PR updated. PTAL.

Reply:
Finally fix all the error messages in the right way: avoiding leaking internal function names. Thanks!